lib/sdp/simple_search.rb

```diff
@@ -0,0 +1,61 @@
+module SDP
+  module SimpleSearch
+    def self.search(type, query_string, current_user_id = nil, limit = 20, page = 1)
+      types = type ? [type_to_class(type)] : [Form, Question, ResponseSet]
```

Comment:
I think if type is blank or nil here, there's a null dereference later.

You could remove type_to class() and replace this line with something like
      type &&= type.camelize
      types = %w(Form Question ResponseSet).include?(type) ? [type.constantize] : [Form, Question, ResponseSet]

I totally spent 20 minutes code golfing this.